html/browsers/the-window-object/self-et-al.window.js

```diff
@@ -0,0 +1,26 @@
+[
+  "frames",
+  "globalThis",
+  "self",
+  "window"
+].forEach(windowProxySelfReference => {
+  test(() => {
+    const frame = document.body.appendChild(document.createElement("iframe")),
+          otherW = frame.contentWindow;
+    assert_equals(otherW[windowProxySelfReference], otherW);
+    frame.remove();
+    assert_equals(otherW[windowProxySelfReference], otherW);
```

Comment:
Adding assertion text to these asserts would help, e.g. right now https://wpt.fyi/results/html/browsers/the-window-object/self-et-al.window.html?diff&filter=ADC&run_id=6201193340600320&run_id=5327201419395072 is not very informative.